app/assets/javascripts/comment.js

```diff
@@ -0,0 +1,12 @@
+// Place all the behaviors and hooks related to the matching controller here.
+// All this logic will automatically be available in application.js.
+// You can use CoffeeScript in this file: http://coffeescript.org/
+$(document).ready(function() {
```

Comment:
```js
$(document).ready()
```
This also fails when using Turbolinks. 
http://stackoverflow.com/questions/18770517/rails-4-how-to-use-document-ready-with-turbo-links